test/web/10-CredentialCard.js

```diff
@@ -11,19 +11,55 @@ function renderCredentialCard(propsData) {
 }
 
 describe('CredentialCard', () => {
-  it('should check something on the rendered element', async () => {
-    // TODO: specify propsData
+  it('should render content if type of value is not falsy in credential' +
+    'subject', async () => {
     const vm = renderCredentialCard({
       credential: {
-        name: 'Test Credential',
+        credentialSubject: {
+          birthCountry: 'Mexico',
+        },
         type: ['TestCredential'],
         description: 'Test description',
         image: 'http://example.com/some-image.png'
       },
-      schema: {}
+      schema: {
+        birthCountry: {
+          name: 'Birth Country'
+        }
+      }
     });
     should.exist(vm);
     should.exist(vm.$el);
-    // TODO: $vm.el.something
+    vm.$el.querySelector(
+      '.q-item__section.g-field-data-regular.q-item__section--main')
+      .textContent.trim().should.equal('Mexico');
+  });
+
+  it('should not render content if type of value is falsy in credential' +
+    'subject', async () => {
+    const valueTypes = ['', NaN, 0, null, false, undefined];
+
+    for(const value of valueTypes) {
+      const vm = renderCredentialCard({
+        credential: {
+          credentialSubject: {
+            birthCountry: value,
+          },
+          type: ['TestCredential'],
+          description: 'Test description',
+          image: 'http://example.com/some-image.png'
+        },
+        schema: {
+          birthCountry: {
+            name: 'Birth Country'
```

Comment:
this actually is an object with a name and an icon such as:

https://github.com/digitalbazaar/bedrock-vue-credential-card/blob/c86cc0b21b8ecae6ae9d6d146520d01db33c032e/test/components/Home.vue#L123-L141

I really hate to say this, but I'm not sure if the icon is optional or not.

EDIT: I think it is working with out an icon so that's fine.